BaseFilter/sections/specific.txt

```diff
@@ -566,6 +566,13 @@ $subdocument,third-party,denyallow=disqus.com,domain=welovemanga.one
 ! NOTE: Specific rules
 !
 !
+ww2-weapons.com###sgpb-popup-dialog-main-div-wrapper
+ww2-weapons.com##.sgpb-theme-6-overlay
+ww2-weapons.com##.widget.warhist-widget
+ww2-weapons.com###nav_menu-2
+ww2-weapons.com###media_image-2
+ww2-weapons.com###media_image-3
+ww2-weapons.com##div[class="warhist-after-content"]
```

Comment:
may be more general
`ww2-weapons.com##[class*="warhist-"]`

Reply:
Suggested `div[id^="warhist-"]`